aspnet/data/entity-framework-6.rst

```diff
@@ -1,110 +1,97 @@
-    :linenos:
-    
-    <entityFramework>
-        <defaultConnectionFactory type="System.Data.Entity.Infrastructure.LocalDbConnectionFactory, EntityFramework">
-            <parameters>
-                <parameter value="mssqllocaldb" />
-            </parameters>
-        </defaultConnectionFactory>
-        <providers>
-            <provider invariantName="System.Data.SqlClient" type="System.Data.Entity.SqlServer.SqlProviderServices, EntityFramework.SqlServer" />
-        </providers>
-    </entityFramework>
-
-The ``defaultConnectionFactory`` element sets the factory for connections. If this attribute is not set then the default value is ``SqlConnectionProvider``. If, on the other hand, value is provided, the given class will be used to create ``DbConnection`` with its ``CreateConnection`` method. If the given factory has no default constructor then you must add parameters that are used to construct the object.
-
-.. code-block:: c#
-    :linenos:
-
-    [DbConfigurationType(typeof(CodeConfig))] // point to the class that inherit from DbConfiguration
-    public class ApplicationDbContext : DbContext
-    {
-        [...]
-    }
-    
-    public class CodeConfig : DbConfiguration
-    {
-        public CodeConfig()
-        {
-            SetProviderServices("System.Data.SqlClient",
-                System.Data.Entity.SqlServer.SqlProviderServices.Instance);
-        }
-    }
-    
-SQL Server, SQL Server Express and LocalDB
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. literalinclude::  entity-framework-6/sample/EF6/SchoolContextFactory.cs
+  :language: c#
+  :start-after: snippet_IDbContextFactory
+  :end-before:  #endregion
+  :dedent: 4
+
+In this sample code, the ``IDbContextFactory`` implementation passes in a hard-coded connection string. This is the connection string that the command-line tools will use. You'll want to implement a strategy to ensure that the class library uses the same connection string that the calling application uses. For example, you could get the value from an environment variable in both projects.
+   
+Set up dependency injection in the ASP.NET Core project
+-------------------------------------------------------
 
-This is the default and so no explicit configuration is needed. The above ``CodeConfig`` class can be used to explicitly set the provider services and the appropriate connection string should be passed to the ``DbContext`` constructor as shown `above <#setup-connection-strings-and-dependency-injection>`_.
+In the Core project's *Startup.cs* file, set up the EF6 context for dependency injection (DI) in ``ConfigureServices``. EF context objects should be scoped for a per-request lifetime.
+
+.. literalinclude::  entity-framework-6/sample/MVCCore/Startup.cs
+  :language: c#
+  :start-after: snippet_ConfigureServices
+  :end-before:  #endregion
+  :dedent: 8
+
+You can then get an instance of the context in your controllers by using DI the same way you would for an EF Core context.
```

Comment:
Nit: I don't think we should say it is done in the same way but in a *similar* way, e.g. we don't have `AddDbContext<TContext>()` for EF6.